docs/CONTRIBUTING.md

```diff
@@ -138,6 +138,17 @@ Successfully installed apipkg-1.4 attrs-18.1.0 bandersnatch-2.1.3 coverage-4.5.1
 /path/to/venv/bin/pip install -e .
 ```
 
+## Creating a Pull Request
+
+### Changelog entry
+
+PRs must have an entry in CHANGES.md that references the PR number in the format of
+"PR #{number}". Yes, you'll either have to guess your PR number or do another push
+after the fact.
```

Comment:
I thought about adding a justification for the additional work that is now required. but I wasn't able to come up with something that sounded fair to all parties.